fhir-persistence-schema/src/main/java/com/ibm/fhir/schema/derby/DerbyFhirDatabase.java

```diff
@@ -40,6 +40,7 @@
     private static final String SCHEMA_NAME = "FHIRDATA";
     private static final String ADMIN_SCHEMA_NAME = "FHIR_ADMIN";
     private static final String OAUTH_SCHEMANAME = "FHIR_OAUTH";
+    private static final String BATCH_SCHEMANAME = "FHIR_BATCH";
```

Comment:
better to change to FHIR_JBATCH

Reply:
and are those constants common for derby, db2 etc.

Reply:
thanks for seeing this... I am trying to use JBATCH (good find)

Reply:
Fixed in next PR.